django_lightweight_queue/middleware/transaction.py

```diff
@@ -1,30 +1,24 @@
-from django.db import transaction, connection
+import warnings
+
+from django.db import transaction
 
 class TransactionMiddleware(object):
+    def _raise_deprecation_warning(self):
```

Comment:
I'm not sure I understand why this was needed?

Is there anything which prevents the middleware having:
``` python
def is_atomic_task(self, job):
    return job.get_task_instance().atomic
```

and then keeping the transaction condition in the middleware?

To my mind, this is a cleaner solution as it keeps the logic in the middleware (though I can see the disadvantage that a user could define a `@task(atomic=True)` and fail to include the middleware, however that's always the case with anything provided by middleware).